app/controllers/backgrounds_controller.rb

```diff
@@ -25,7 +25,10 @@ def edit
   # POST /backgrounds
   # POST /backgrounds.json
   def create
-    @background = Background.new(background_params)
+    @background = Background.create(background_params)
```

Comment:
shouldn't this be inside the transaction if we want the whole thing to rollback if the creation failed?

Reply:
absolutely, thanks